### ISSUE TITLE
Hiding a game doesn't disable the game in the Pinball X database files

### ISSUE THREAD

**Issue by rtl19** [mjrgh/PinballY#103]
When using the Pinball Y frontend to modify games that are contained in the Pinball X database files, the act of hiding the game does not mark the xml field for "enabled" to False.  When going back to Pinball X, duplicate games are present when they should in fact be disabled.

**Comment by mjrgh** [comment]
I just gave it a try, and it seems to be working for me.  Have you looked at the XML file?

**Comment by rtl19** [comment]
Glad to hear what I saw isn't expected behavior!  The one or two tables that I hid still showed as enabled in the XML.  I was doing alot at the time so I'll conduct a targeted experiment using the latest beta to confirm I'm still seeing the issue.

**Comment by rtl19** [comment]
Good news.  Hiding is also disabling the table in the XML.  I'm using Beta 18.  Also, before doing my targeted tests, I went through my 3 XML files and manually removed any possible duplicates and anything that was disabled already.  I then picked a table and hid it.  After going back into the XML I saw the <enabled> field was False.  The only thing I can think of was that I had duplicates of some tables and perhaps misread the duplicate as enabled because it was newly discovered.  Probably my human error.
